0-bootstrap/hub/2-services/argocd/instances/openldap.yaml

```diff
@@ -160,7 +160,7 @@ spec:
             sn: cpadmin
             uid: cpadmin
             mail: cpadmin@cp.local
-            userpassword:: {{ universal_password | b64encode }}
+            userpassword: Not@SecurePassw0rd
```

Comment:
I take it it is up to the openldap admin to change the passwords for these initial users during app run time - I'm ok with that.


Since they all have the same password - will the ldap app throw errors re duplicate passwords? If so, may be a good idea to change each slightly different to one another.